LIGHTNING-00-install.md

```diff
@@ -1,13 +1,16 @@
+[ [index](/README.md), [next](/LIGHTNING-01-peers.md) ]
+
 # Lightning
-[Lightning Installation Guide](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md) is used to build the dependencies (`Go`), currency daemons (`btcd` and `ltcd`) and the actual `lnd` daemon.  
+This guide follows [Lightning Installation Guide](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md) to build the dependencies (`Go`), currency daemons (`btcd` and `ltcd`) and the actual `lnd` daemon.  
 
 ## Install Lightning Dependencies
-Download latest package from [offical Go repository](https://golang.org/dl/) and uncompress to `/usr/local`
+Download latest (`Go`) package from [offical Go repository](https://golang.org/dl/) and uncompress to `/usr/local` .
+Alternativly you can use apt-get
 ```shell
-$ sudo tar -C /usr/local -xzf go1.9.2.linux-amd64.tar.gz
+$ sudo apt-get install golang-1.10-go
```

Comment:
Just a minor detail but you need to add a apt repository before you can install Go version 1.10. What about adding [a link to the documentation](https://github.com/golang/go/wiki/Ubuntu) instead of the commands themself?

Reply:
Two lines above there is a reference + link to the Lightning installation guide. Not sure there is a need to add another link.